notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-095_settling_time_rms.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>Comment says that 2 plots are shown but I only see one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>